tests/ui/manual_unwrap_or.rs

```diff
@@ -80,4 +80,61 @@ fn unwrap_or() {
     };
 }
 
+fn result_unwrap_or() {
+    // int case
+    match Ok(1) as Result<i32, &str> {
```

Comment:
Not sure what the difference is in this case, but I've seen this more commonly written using turbofish syntax than using an `as` cast
```rust
Ok::<i32, &str>(1)
```

Reply:
I didn't know the turbofish syntaxt nor its cool name. It looks indeed more concise, I'll change that.

Reply:
If you like the name, don't miss [The Bastion of the Turbofish](https://github.com/rust-lang/rust/blob/79d8a0fcefa5134db2a94739b1d18daa01fc6e9f/src/test/ui/bastion-of-the-turbofish.rs) then ;)